ext/gd/tests/imagexbm_nullbyte_injection.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Testing null byte injection in imagexbm
+--SKIPIF--
+<?php
+if(!extension_loaded('gd')) die('skip gd extension not available');
```

Comment:
May need more skipif?

Reply:
Not for XBM, but indeed for BMP. Thanks! And actually we can drop the check for GIF support.

Reply:
Done.